azurelinuxagent/common/protocol/restapi.py

```diff
@@ -111,6 +113,15 @@ def __init__(self, name=None):
         self.name = name
         self.properties = ExtHandlerProperties()
         self.versionUris = DataContractList(ExtHandlerVersionUri)  # pylint: disable=C0103
+        self.__invalid_handler_reason = None
+
+    @property
+    def is_invalid_with_reason(self):
```

Comment:
Introduced a new property to check if the ExtHandler is invalid for any reason. 
The benefit of adding this here is - 
- If extA is invalid but extB is valid, we would still execute both extA and extB and report faulty handler for only extA. 

(Note: Open to suggestions on the name of the property. Couldn't figure out a better name for what its trying to accomplish)